### PR TITLE
Release v6.0.0

### PR DESCRIPTION
**Pull request by str4d**

### CODE REVIEW

**Review by daira — DISMISSED** [review]
utACK. Double-checked the FPF address, funding stream end height, and NU6 mainnet network protocol version. Needs consensus and release review (ideally 3 acks).

File: src/wallet/test/rpc_wallet_tests.cpp [hunk 1]
```diff
@@ -315,8 +315,25 @@ BOOST_AUTO_TEST_CASE(rpc_wallet)
         }
     };
 
+    auto check_lockbox_streams = [](UniValue obj, std::vector<std::string> recipients, std::vector<double> amounts) {
+        size_t n = recipients.size();
+        BOOST_REQUIRE_EQUAL(amounts.size(), n);
+        UniValue lockboxstreams = find_value(obj, "lockboxstreams");
+        BOOST_CHECK_EQUAL(lockboxstreams.size(), n);
+        if (lockboxstreams.size() != n) return;
```
Comment (daira) [inline src/wallet/test/rpc_wallet_tests.cpp:323]: Philosophical conundrum: does the difference between `BOOST_REQUIRE` and `BOOST_CHECK` reflect semantics ("the test requires this" vs "the test is intended to check this"), or just the practical issue that the test shouldn't proceed past a failed `BOOST_REQUIRE`? If it's the former then this is correct. If the latter then it's overcomplicated and could just be `BOOST_REQUIRE_EQUAL(lockboxstreams.size(), n);`. But I'm fine with it as-is.

File: src/wallet/test/rpc_wallet_tests.cpp [hunk 2]
```diff
@@ -346,9 +364,27 @@ BOOST_AUTO_TEST_CASE(rpc_wallet)
...
+                                   { 0.125,                       },
+                                   {
+                                       "t3cFfPt1Bcvgez9ZbMBFWeZsskxTkPzGCow"
+                                   });
+        check_lockbox_streams(obj, { "Lockbox NU6" },
+                                   { 0.1875,       });
+    } else {
+        BOOST_CHECK(find_value(obj, "fundingstreams").empty());
+        BOOST_CHECK(find_value(obj, "lockboxstreams").empty());
+    }
```
Comment (daira) [inline src/wallet/test/rpc_wallet_tests.cpp:384]: I ❤️ your thoroughness.

**Review by nuttycom — APPROVED** [review]
utACK 26939a9710ca3d467d375291b0bbfedf6a2ce26f

Thank you both for the effort to get this to the finish line while I slept!

**Review by nuttycom — APPROVED** [review]
re-utACK c61e5473ae1fee18ac57e14123d3943323f0f04c

**Review by arya2 — APPROVED** [review]
utACK, double-checked the FPF address, funding stream height ranges, funding periods, current and NU6 network protocol versions, NU6 activation height, and the end-of-support halt height/date.

**Comment by daira** [comment]
Confirmed to tag c61e5473ae1fee18ac57e14123d3943323f0f04c as the release.

**Pull request merged**